docs/pages/selectlist.js

```diff
@@ -1,39 +1,69 @@
 // @flow strict
 import { type Node } from 'react';
+import AccessibilitySection from '../components/AccessibilitySection.js';
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import MainSection from '../components/MainSection.js';
 import Page from '../components/Page.js';
 import PageHeader from '../components/PageHeader.js';
-import MainSection from '../components/MainSection.js';
-import GeneratedPropTable from '../components/GeneratedPropTable.js';
-import docgen, { overrideTypes, type DocGen } from '../components/docgen.js';
 import QualityChecklist from '../components/QualityChecklist.js';
+import { multipledocgen, type DocGen } from '../components/docgen.js';
 
-import AccessibilitySection from '../components/AccessibilitySection.js';
-
-export default function SelectListPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+export default function DocsPage({
+  generatedDocGen,
+}: {|
+  generatedDocGen: {| [string]: DocGen |},
+|}): Node {
   return (
-    <Page title="SelectList">
+    <Page title={generatedDocGen?.SelectList?.displayName}>
       <PageHeader
-        name="SelectList"
-        description={generatedDocGen?.description}
+        name={generatedDocGen?.SelectList?.displayName}
+        description={generatedDocGen?.SelectList?.description}
         defaultCode={`
 <SelectList
   id="selectlistexample1"
+  label="Country"
   onChange={() => {}}
-  options={[
-    {label:'Algeria', value: 'algeria'},
-    {label:'Belgium', value: 'belgium'},
-    {label:'Canada', value: 'canada'},
-    {label:'Denmark', value: 'denmark'},
-    {label:'Egypt', value: 'egypt'},
-    {label:'France', value: 'france'},
-  ]}
-  size='lg'
-  label='Country'
-/>
+  size="lg"
+>
+  {[
```

Comment:
In the future we can update some/all of these examples to use the more HTML-ish approach, if desired. This is the quick+easy version for now

_HTML-ish approach_
```jsx
<SelectList>
  <SelectList.Option label="Algeria" value="algeria" />
  <SelectList.Option label="Belgium" value="belgium" />
  ...
</SelectList>
```